### PR TITLE
New Package: compfy-1.7.1

### PR DESCRIPTION
**Pull request by saltnpepper97**
New Package: compfy-1.7.1

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by saltnpepper97** [comment]
I apologize for the adding and removing this commit yesterday. I realize now that it was a bad idea. Here is my final upload of compfy as a new package for void linux. I think it would be a great addition to the package list as it is essentially Picom but more actively maintained by allusive-dev and has animations included as well as several fixes to base picom.

File: srcpkgs/compfy/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'compfy'
+pkgname=compfy
+version=1.7.1
+revision=2
```
Comment (abenson) [inline srcpkgs/compfy/template:4]: Revision should be set to 1.

**Comment by abenson** [comment]
The commits need to be [squashed](https://www.git-tower.com/learn/git/faq/git-squash) into a single "New package..." commit. Once that is done, force push to this branch: `git push -f`.

**Comment by abenson** [comment]
Whoops, did not mean to close that, sorry.

**Comment by saltnpepper97** [comment]
Fixed the revision number and added conflicts for picom and compton respectively.

**Comment by saltnpepper97** [comment]
Squashed it as well. You are welcome!

File: srcpkgs/compfy/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'compfy'
+pkgname=compfy
+version=1.7.1
+revision=1
+build_style=meson
+configure_args="-Dupdate_checks=true"
```
Comment (abenson) [inline srcpkgs/compfy/template:6]: Typically we disable update checking in packages.

Comment (saltnpepper97) [reply srcpkgs/compfy/template]: Removed the check updates.

Comment (saltnpepper97) [reply srcpkgs/compfy/template]: anything else needed??

File: srcpkgs/compfy/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+hostmakedepends="pkg-config asciidoc"
+makedepends="libconfig-devel dbus-devel libev-devel libglvnd-devel libX11-devel libxcb-devel libxdg-basedir-devel pcre2-devel pixman-devel uthash xcb-util-image-devel xcb-util-renderutil-devel json-c-devel libcurl-devel"
+depends="desktop-file-utils"
+short_desc="X11 compositor made as an alternative to Picom with animations"
+maintainer="Dustin Pilgrim <dustin.pilgrim1997@gmail.com>"
+license="MIT, MPL-2.0"
+homepage="https://github.com/allusive-dev/compfy"
+distfiles="https://github.com/allusive-dev/compfy/archive/refs/tags/${version}.tar.gz"
+checksum=7c41068ca6ab737c6b6fca453bb5b69a3f94c1de45a46daf79ec469f32b626ed
+comflicts="compton picom"
```
[the remark this answers is not in the document]
Comment (saltnpepper97) [reply srcpkgs/compfy/template]: Fixed as well. Sorry for the typos XD

**Pull request closed without merging**